docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,7 +2,7 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate commands to devfiles v2.x. The command section of devfile v2.x has major changes. A command specified in a devfile v1.x no longer works in a devfile v2.x.
+This section describes how to migrate commands to devfiles v2.x. Major changes have been made from devfiles v1.x to the command section of devfile v2.x. A command specified in a devfile v1.x no longer works in a devfile v2.x.
```

Comment:
"Major changes have been made from devfiles v1.x to the command section of devfile v2.x"
How about you delete this sentence. I can't think of a good way to say it. The meaning you are going for is this, right? "Compared to devfile v1.x, devfile v2.x is very different"? If so, I don't think this is strictly necessary information, since you mention immediately afterwards that a command specified in a devfile v1.x no longer works in a devfile v2.x.
WDYT?

Reply:
Agreed. I think the original intent was to communicate that the new version of devfile has major changes compared to version 1. 

But now I think we mostly want to communicate if devfile v1.x still works or does not work in devfile v2.x. We'll communicate that message by keeping the last sentence!

Reply:
Resolving this conversation as I have made the suggested change.